mipengine/controller/api/error_handlers.py

```diff
@@ -1,22 +1,35 @@
+import traceback
+
 from quart import Blueprint
 
 from mipengine.controller.api.exceptions import BadRequest
 from mipengine.controller.api.exceptions import BadUserInput
-from mipengine.controller.api.exceptions import UnexpectedException
+from mipengine.node_tasks_DTOs import PrivacyError
 
 error_handlers = Blueprint("error_handlers", __name__)
 
+PRIVACY_ERROR_MESSAGE = "The algorithm could not run with the input provided because there are insufficient data."
+
 
 @error_handlers.app_errorhandler(BadRequest)
 def handle_bad_request(error: BadRequest):
-    return error.message, error.status_code
+    return error.message, 400
```

Comment:
could we add an Enum for the http error codes, denoting what they actually mean, so we do not have numbers floating around in the code? Especially since some of these are custom error codes denoting a specific error like the 461, something like:
```
class HTTPStatusCode(enum.Enum):
    400="BadRequest"
    460="BadUserInput"
    461="PrivacyError"
    500="UnexpectedError"
```